docs/scos/dev/glue-api-guides/202304.0/dynamic-data-api/how-to-guides/how-to-send-request-in-dynamic-data-api.md

```diff
@@ -0,0 +1,456 @@
+To apply filtering to the results based on specific fields, include the appropriate filter parameter in the request:  
+
+```bash
+GET /dynamic-entity/country?filter[country.iso2_code]=AA HTTP/1.1
+Host: glue-backend.mysprykershop.com
+Content-Type: application/json
+Accept: application/json
+Authorization: Bearer {your_token}
+```
+
+{% info_block warningBox "Verification" %}
+
+If everything is set up correctly you will get the following response:
+
+```json
+[
+  {
+    "id_country": 1,
+    "iso2_code": "AA",
+    "iso3_code": "UUD",
+    "name": "Create",
+    "postal_code_mandatory": false,
+    "postal_code_regex": null
+  }
+]
+```
+
+{% endinfo_block %}
+
+{% info_block infoBox %}
+
+Note, so far when you combine multiple filters in a single request, the system applies an "AND" condition to the retrieved results.
+
+{% endinfo_block %}
+
+To retrieve a country by a specific ID, you can send a `GET` request with the following parameters:
+
+```bash
+GET /dynamic-entity/country/3 HTTP/1.1
+Host: glue-backend.mysprykershop.com
+Content-Type: application/json
+Accept: application/json
+Authorization: Bearer {your_token}
+```
+{% info_block warningBox "Verification" %}
+
+If everything is set up correctly you will get the following response:
+
+```json
+[
+  {
+    "id_country": 3,
+    "iso2_code": "AE",
+    "iso3_code": "ARE",
+    "name": "United Arab Emirates",
+    "postal_code_mandatory": false,
+    "postal_code_regex": null
+  }
+]
+```
+
+{% endinfo_block %}
+
+{% info_block infoBox %}
+
+Note if a current endpoint is not configured in `spy_dynamic_entity_configuration` and it's not found you'll get the following response:
+
+```json
+[
+    {
+        "message": "Not found",
+        "status": 404,
+        "code": "007"
+    }
+]
+```
+
+{% endinfo_block %}
+
+### Sending `POST` request
+
+To create a collection of countries, submit the following HTTP request:
+
+```bash
+POST /dynamic-entity/country HTTP/1.1
+Host: glue-backend.mysprykershop.com
+Content-Type: application/json
+Accept: application/json
+Authorization: Bearer {your_token}
+Content-Length: 154
+
+{
+    "data": [
+        {
+            "iso2_code": "WA",
+            "iso3_code": "WWA",
+            "name": "FOO"
+        }
+    ]
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+If everything is set up correctly you will get the following response:
+
+```json
+[
+  {
+    "iso2_code": "WA",
+    "iso3_code": "WWA",
+    "name": "FOO",
+    "id_country": 257
+  }
+]
+```
+
+The response payload includes all the specified fields from the request body, along with the id of the newly created entity.
+
+{% endinfo_block %}
+
+{% info_block infoBox %}
+
+Note that all fields included in the request payload are marked as `isCreatable: true` in the configuration. 
+Therefore, the API will create a new record with all the provided fields. 
+However, if any of the fields are configured as `isCreatable: false` it won't be created. 
```

Comment:
It is not always true: if it is `isCreatable: false`, but not required, the row will be created.